tests/qa/RS_VERSIONS

```diff
@@ -1,2 +1 @@
-6.0.12-58
 100.0.0-2400
```

Comment:
This should be 100.0.0-2440 isn't it? @tomerhekredis ?

Reply:
As above.